python/pyproject.toml

```diff
@@ -24,7 +24,10 @@ description = "Provides a tag-expression parser and evaluation logic for cucumbe
 version = "6.2.0"
 # PREPARED: dynamic = ["version"]
 keywords= ["BDD", "testing", "cucumber", "tag-expressions", "behave"]
-license = {text = "MIT"}
+license = "MIT"
+license-files = [
+     "../LICENSE"
```

Comment:
For consistency, you can use the `LICENSE` file in the `python` directory. Most other build tools can't reach into parent directories for extra files so we don't have to do that here either.

This applies to both of your PRs.